improver_tests/utilities/test_cube_extraction.py

```diff
@@ -407,6 +399,82 @@ def test_range_constraints(self):
         reference_data = self.precip_cube.data[:2, :, :]
         self.assertArrayEqual(cube.data, reference_data)
 
+    def test_subset_uk_grid(self):
+        """ Test subsetting a gridded cube. """
+        expected_data = np.array(
+            [
+                [27.0, 28.0, 29.0, 30.0],
+                [35.0, 36.0, 37.0, 38.0],
+                [43.0, 44.0, 45.0, 46.0],
+                [51.0, 52.0, 53.0, 54.0],
+            ]
+        )
+        expected_points = np.array([1000.0, 3000.0, 5000.0, 7000.0])
+        lower_bound = 1000 * (1.0 - 1.0e-7)
+        upper_bound = 7000 * (1.0 + 1.0e-7)
```

Comment:
I largely copied the code from the tests from the previous `test_subset_cube` so that it'd be clear that the functionality hadn't been removed, I'm not sure why it was written that way initially, I'm happy to change it though.

Reply:
Thanks, I've simplified this for the `test_subset_global_grid_pacific` test too in https://github.com/BelligerG/improver/pull/4.